### ISSUE TITLE
Request -- support for arm64 architecture

### ISSUE THREAD

**Issue by EtaoinWu** [aiogram/telegram-bot-api#9]
The docker image currently only has `linux/386` and `linux/amd64` variants. Is it possible that you can build the same image for `linux/arm64/v8`?

**Comment by Luoxin** [comment]
> The docker image currently only has `linux/386` and `linux/amd64` variants. Is it possible that you can build the same image for `linux/arm64/v8`?

Mabey you can build use

```bash
git clone --recursive https://github.com/aiogram/telegram-bot-api.git
cd telegram-bot-api
make build
```

**Comment by JrooTJunior** [comment]
Of course, you can build this image manually on any platform using this command.

Right now I already have a draft of the redesigned multiplatformDocker Image (build each platform separatelly and then merge their metadata), I need some free time to finish it, maybe I'll find time for it before the new year, I hope so.

**Comment by rafaelslopes1** [comment]
Do we have any news about this?

**Comment by JrooTJunior** [comment]
Very hard december =(

**Comment by tshipenchko** [comment]
As a workaround, I made [a fork with ARM builds](https://github.com/tshelter/telegram-bot-api) on my own github actions runner.

**Comment by rafaelslopes1** [comment]
> Como solução alternativa, fiz [um fork com compilações ARM](https://github.com/tshelter/telegram-bot-api) em meu próprio executor de ações do github.

Oh, thank you. This will be of great help.

**Comment by rafaelslopes1** [comment]
> As a workaround, I made [a fork with ARM builds](https://github.com/tshelter/telegram-bot-api) on my own github actions runner.

Oh, thank you. This will be of great help.

**Comment by JrooTJunior** [comment]
ARM builds should be available after #12 is merged into the main branch.

**Comment by JrooTJunior** [comment]
Congratulations 🎉🎉🎉

From now you can use ARM builds from Docker Hub

![SmartSelect_20240106_155130_Chrome](https://github.com/aiogram/telegram-bot-api/assets/6726423/72244900-762f-4e2a-9f7d-b5af6689b80f)
